error_report/sentry.py

```diff
@@ -229,6 +230,12 @@ def create_sentry_report(report):
         # group issues by the module of the last frame
         # (unless the last frame is too general)
         data["fingerprint"] = [module]
+    info_msg = "Sentry report created."
+    try:
```

Comment:
Why not
if "Exception" in report:
  info_msg += " Exception: {}".format(report["Exception"])

Reply:
Ok.